source/MaterialXContrib/MaterialXNode/CreateMaterialXNodeCmd.cpp

```diff
@@ -34,67 +35,39 @@ const char* const kTextureFlagLong  = "asTexture";
 const char* const kOgsXmlFlag       = "x";
 const char* const kOgsXmlFlagLong   = "ogsXml";
 
-void registerFragment(const MaterialXData& materialData, const std::string& ogsXmlFileName)
+void registerDebugFragment(const std::string& ogsXmlFileName)
```

Comment:
Separated the two cases (explicit debug fragment read from file and actual generated fragment passed as a string) because the former is only ever executed from the command while the latter can also be executed when attributes are edited on the node after creation.